DomainContent/AvatarStore/attachmentItemScript.js

```diff
@@ -35,6 +35,20 @@
     var firstGrab = true;
 
     var attachDistance = ATTACH_DISTANCE;
+
+    var attachFunction = function(){
+        var posEye = MyAvatar.getEyePosition();
+        var posFoot = MyAvatar.getJointPosition("LeftFoot");
+
+        if (posEye == {x: 0, y:0, z:0} ||
+            posFoot == {x: 0, y:0, z:0} || !HMD.active){
+            return;
+        } else {
+            var estimateHeight = posEye.y - posFoot.y;
+            attachDistance = estimateHeight/3;
+        }
```

Comment:
if else statement could be simplified to be only an if statement, since it is the last statement in the function.